data_structures/binary_tree/avl_tree.py

```diff
@@ -31,18 +31,22 @@ def pop(self):
     def count(self):
         return self.tail - self.head
 
-    def print(self):
+    def print(self) -> None:
```

Comment:
This is a bad method name because it shadows a Python builtin function.  Why not rename this to __str__() and then just `print(my_queue_instance)`?

Reply:
Do you mean __str__ ?

Reply:
https://docs.python.org/3/reference/datamodel.html?highlight=__str__#object.__str__